### DISCUSSION TITLE
Consider changing to a more legible font in the document?

### DISCUSSION THREAD

**Discussion by LittleSound** [vuejs/pinia#738] [Ideas]
![image](https://user-images.githubusercontent.com/19204772/138385924-ef0bf0d3-a487-4340-9f34-cc1cbd019dce.png)

[pinia.esm.dev](https://pinia.esm.dev/introduction.html#a-more-realistic-example)
Some of this text, such as `s`, is difficult to read. Is a change of font being considered?

**Comment by posva** [comment]
This is completely subjective...

**Reply by baixiaoyu2997** [reply]
for non-native english people，like me， it's not friendly 。hard to know  what letter this is。

**Reply by ghost** [reply]
you can try this in violentmonkey
```js
// ==UserScript==
// @name        change pinia code's fonts
// @namespace   Violentmonkey Scripts
// @match       https://pinia.vuejs.org/*
// @grant       none
// @version     1.0
// @author      ruinb0w
// @description 3/8/2022, 11:48:46 AM
// ==/UserScript==

function changeFont() {
  const code_list = document.querySelectorAll("code");
  code_list.forEach((item) => {
    item.style.fontFamily = "monospace";
  });
}

const _wr = function (type) {
  const orig = history[type];
  return function () {
    console.log('arguments', arguments)
    const rv = orig.apply(this, arguments);
    const e = new Event(type);
    e.arguments = arguments;
    window.dispatchEvent(e);
  };
};
history.pushState = _wr("pushState");
history.replaceState = _wr("replaceState");

window.addEventListener("load", changeFont);
window.addEventListener("hashchange", changeFont)
window.addEventListener("replaceState", changeFont);
window.addEventListener("pushState", changeFont);
```

**Reply by RedShift1** [reply]
> This is completely subjective...

ɨ ǟɢʀɛɛ, ʟɛɢɨɮʟɛ ȶɛӼȶ ɨֆ ƈօʍքʟɛȶɛʟʏ ֆʊɮʝɛƈȶɨʋɛ

**Comment by Saman-Safaei** [comment]
I want this font ... But I don't know it's name

**Reply by dehood** [reply]
That makes us two!

**Reply by skuhnow** [reply]
Three

**Reply by tintin10q** [reply]
did anyone ever find the name?

**Reply by qunm00** [reply]
https://philpl.gumroad.com/l/dank-mono 
Here you go. Recommend you guys to look into https://input.djr.com/

**Comment by jimoquinn** [comment]
The e is odd:
![ell](https://user-images.githubusercontent.com/1494571/190511054-c4c335b3-13af-47b8-a7cd-a0c68c6822ca.png)

This one is also odd.
![essess](https://user-images.githubusercontent.com/1494571/190511023-a02bd82a-63ad-4d35-94f9-ec7ca310af72.png)

Not too odd, but I imagine it can be confusing.
![eff](https://user-images.githubusercontent.com/1494571/190511061-43fa01a8-a3cc-40b1-aeab-f390f4f2652b.png)
